packages/quantic/force-app/main/default/lwc/caseClassification/caseClassification.js

```diff
@@ -0,0 +1,195 @@
+import { LightningElement, api } from 'lwc';
+import caseClassificationTitle from '@salesforce/label/c.quantic_CaseClassificationTitle';
+import selectOption from '@salesforce/label/c.quantic_SelectOption';
+import selectAnotherFeature from '@salesforce/label/c.quantic_SelectAnotherFeature';
+import selectProduct from '@salesforce/label/c.quantic_SelectProduct';
+
+/**
+ * A section for a user to classify his case aided by suggestions provided by Coveo Case Assist. There is also a dropdown available to see all available values for a given category.
+ */
+export default class CaseClassification extends LightningElement {
+  labels = {
+    caseClassificationTitle,
+    selectAnotherFeature,
+    selectOption,
+    selectProduct
+  };
+
+  /**
+   * All the options proposed to the user to choose from.
+   * @type {Array}
+   * @defaultValue `[]`
+   */
+  @api options = [];
+
+  /**
+   * The number of suggestions to be shown.
+   * @type {number}
+   * @defaultValue `3`
+   */
+  @api numberOfSuggestions = 3;
+
+  /**
+   * Tells if the case classification input is required.
+   * @type {boolean}
+   * @defaultValue `false`
+   */
+  @api required = false;
+
+  /**
+   * The label to be shown to the user.
+   * @type {string}
+   * @defaultValue `'Which product is related to your problem?'`
+   */
+  @api label = this.labels.caseClassificationTitle;
+
+  /**
+   * The place holder of the combo box input. 
+   * @type {string}
+   * @defaultValue `'Select Product'`
+   */
+  @api slectPlaceHolder = this.labels.selectProduct;
```

Comment:
I'm not sure about using labels related to products by default. Since a developer has to set all these labels every time the component is used, I don't see much value in having the placeholder labels. Perhaps we could add a warning message in the console when required properties are not set properly.